dcs/mapping.py

```diff
@@ -98,6 +98,18 @@ class Vector2:
     y: float
 
 
+@dataclass
+class WWIIFollowBigFormationPosition:
+    x: int
+    y: int
+    z: int
+
+    def __eq__(self, other):
```

Comment:
That doesn't really make the test any more difficult, and while I'm all for making testable APIs, not at the expense of making the implementation incorrect. Like I said, `WWIIFollowBigFormationPosition(0, 0, 0)` and `{"x": 0, "y": 0, "z": 0}` should not map to the same place in a `dict`, but that's what this would do. `dict` might be a bad way to think about it since it's not terribly likely that you'd use a `Vec3` as a dictionary key, but it's quite plausible that you'd want to add these to a `set`, and that would have the same problem.